colobot-base/src/common/version.h.cmake

```diff
@@ -13,5 +13,9 @@ namespace Version
 
     static inline constexpr std::string_view FULL_NAME = "Colobot: Gold Edition @COLOBOT_VERSION_FULL@";
     static inline constexpr std::string_view VERSION = "@CMAKE_PROJECT_VERSION@";
+    static inline constexpr int MAJOR = @CMAKE_PROJECT_VERSION_MAJOR@;
+    static inline constexpr int MINOR = @CMAKE_PROJECT_VERSION_MINOR@;
+    static inline constexpr int PATCH = @CMAKE_PROJECT_VERSION_PATCH@;
+    static inline constexpr int TWEAK = @CMAKE_PROJECT_VERSION_TWEAK@;
```

Comment:
TWEAK should be removed, we will not be using it in future releases.

Reply:
Correct, we had ideas for it but that's unnecessary work for little benefits. Top level project's tweak number will also be removed.